### ISSUE TITLE
How to transpose the following kind of chords?

### ISSUE THREAD

**Issue by thisarasr** [jessegavin/jQuery-Chord-Transposer#12]
I have a chord something like below.

> [Intro]
----------------------------------------------
C | F | C | G
F | C | G7| C
C | F | C | G
F | C | G7| C

Is it possible to transpose this?

**Comment by jessegavin** [comment]
yes, just remove the `|` symbols and you should be able to transpose it

**Comment by thisarasr** [comment]
Yes. That works. But is there a way to add support to transpose mentioned kind types?
